adoc/admin-crio-registries.adoc

```diff
@@ -0,0 +1,58 @@
+== Configuring container registries for {crio}
+
+Every registry related configuration needs to be done in the TOML file
+`/etc/containers/registries.conf`. This system-wide configuration exists in two
+versions, whereas the latest version supports extended features like the
```

Comment:
If we mention there are "two version" then please describe what both of these are and which one is relevant to the user.

Reply:
I will stick to the second version, but we have to check which configuration we ship (I take care of that).